tests/app/submitter/test_converter_v2.py

```diff
@@ -0,0 +1,160 @@
+from datetime import datetime, timezone
```

Comment:
Like my comment about parametrising the tests above. This is a duplicate of test_converter. The only difference is the store is different, which is what should be parametrised.